app/models/user.rb

```diff
@@ -343,6 +343,27 @@ def perform_export_photos!
     update exporting_photos: false
   end
 
+  ######### Data import ##################
+  mount_uploader :import, JsonUploader
+
+  def data_import(importedfile)
+    json_string = importedfile.read()
+    data_hash = JSON.parse(json_string)
+    if self.email != data_hash['user']['email']
```

Comment:
- Style/IfUnlessModifier: Favor modifier `if` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`.
- Style/RedundantSelf: Redundant `self` detected.
- Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.